### PR TITLE
chore: update to Go 1.13

### PR DESCRIPTION
**Pull request by troyronda**
This change updates the project to Go 1.13
- CI is updated.
- README is updated.
- Built-in errors package is used.

Closes: #204

Signed-off-by: Troy Ronda <troy@troyronda.com>

### CODE REVIEW

File: pkg/restapi/restapi_test.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 func TestNew_Failure(t *testing.T) {
 	controller, err := New(&context.Provider{})
 	require.Error(t, err)
-	require.Equal(t, err, api.SvcErrNotFound)
+	require.Equal(t, err, api.ErrSvcNotFound)
```
Comment (troyronda) [inline pkg/restapi/restapi_test.go]: Should I just shorten to ErrNotFound?

Comment (rolsonquadras) [reply pkg/restapi/restapi_test.go]: @troyronda in that case, question is how can we differentiate between different `NotFound` error cases (in same package)? e.g. 'Data not found' and 'Service not found'. Having a specific error message might help in future when we come across these type of scenario's imo.

Comment (llorllale) [reply pkg/restapi/restapi_test.go]: > Should I just shorten to ErrNotFound?

Yes.

Comment (sudeshrshetty) [reply pkg/restapi/restapi_test.go]: > @troyronda in that case, question is how can we going to differentiate between `NotFound` error cases (in same package)? e.g. 'Data not found' and 'Service not found'. Having a specific error message might help in future when we come across these type of scenario's imo.

`ErrSvcNotFound` is good for long run

**Comment by troyronda** [comment]
Created followup regarding ErrNotFound: https://github.com/hyperledger/aries-framework-go/issues/207

**Pull request merged**